server/src/main/java/org/eclipse/openvsx/json/ExtensionJson.java

```diff
@@ -64,12 +64,18 @@ public static ExtensionJson error(String message) {
     @ApiModelProperty(hidden = true)
     public Boolean active;
 
-    @ApiModelProperty("The value 'true' means the extension's namespace is restricted, but the publishing user is not a member of that namespace")
+    @ApiModelProperty("The value 'true' means the publishing user is a member of the extension's namespace and the namespace has at least one owner.")
     @NotNull
-    public boolean unrelatedPublisher;
+    public Boolean verified;
 
-    @ApiModelProperty(value = "Access level of the extension's namespace", allowableValues = "public,restricted")
+    @ApiModelProperty("Deprecated: use 'verified' instead (this property is just the negation of 'verified')")
     @NotNull
+    @Deprecated
+    public Boolean unrelatedPublisher;
```

Comment:
Turning this into `Boolean` doesn't break the API contract, I guess? Is this some sort of clean up?

Reply:
Right, the API contract is defined on a JSON level (see https://open-vsx.org/swagger-ui/). Using wrapper types prevents Jackson from serializing the property when there is no value, which happens in case of error.